symfony/plugins/orangehrmTimePlugin/Api/TimeConfigPeriodAPI.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Time\Api;
+
+use OrangeHRM\Core\Api\CommonParams;
+use OrangeHRM\Core\Api\V2\Endpoint;
+use OrangeHRM\Core\Api\V2\EndpointResourceResult;
+use OrangeHRM\Core\Api\V2\EndpointResult;
+use OrangeHRM\Core\Api\V2\RequestParams;
+use OrangeHRM\Core\Api\V2\ResourceEndpoint;
+use OrangeHRM\Core\Api\V2\Validator\ParamRule;
+use OrangeHRM\Core\Api\V2\Validator\ParamRuleCollection;
+use OrangeHRM\Core\Api\V2\Validator\Rule;
+use OrangeHRM\Core\Api\V2\Validator\Rules;
+use OrangeHRM\Core\Service\MenuService;
+use OrangeHRM\Core\Traits\Service\ConfigServiceTrait;
+use OrangeHRM\Framework\Services;
+use OrangeHRM\Time\Api\Model\TimeConfigPeriodModel;
+use OrangeHRM\Time\Dto\TimeConfigPeriod;
+use OrangeHRM\Time\Service\TimeSheetPeriodService;
+
+class TimeConfigPeriodAPI extends Endpoint implements ResourceEndpoint
+{
+    use ConfigServiceTrait;
+
+    public const PARAMETER_START_DAY = 'startDay';
+
+    /**
+     * @var TimeSheetPeriodService|null
+     */
+    protected ?TimeSheetPeriodService $timeSheetPeriodService = null;
+
+    /**
+     * @return TimeSheetPeriodService
+     */
+    protected function getTimeSheetPeriodService(): TimeSheetPeriodService
+    {
+        if (!$this->timeSheetPeriodService instanceof TimeSheetPeriodService) {
+            $this->timeSheetPeriodService = new TimeSheetPeriodService();
+        }
+        return $this->timeSheetPeriodService;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getOne(): EndpointResult
+    {
+        $status = $this->getTimeSheetPeriodService()->isTimesheetPeriodDefined();
+        $startDay = $status ? $this->getConfigService()->getTimeSheetPeriodConfig() : '2'; // to set monday as default
```

Comment:
`$this->getConfigService()->getTimeSheetPeriodConfig()` this returning xml right?